packages/lab/stories/tabs-next/tabs-next.stories.tsx

```diff
@@ -0,0 +1,142 @@
+import { TabPanelNext, TabsNext, TabsNextProps } from "@salt-ds/lab";
+import { Text } from "@salt-ds/core";
+import { Story } from "@storybook/react";
+import { useState } from "react";
+
+export default {
+  title: "Lab/Tabs Next/Tabs Next",
+  component: TabsNext,
+  args: {
+    activeTabIndex: undefined,
+    onChangeTabIndex: undefined,
+  },
+};
+
+type TabsStory = Story<
+  TabsNextProps & {
+    width?: number;
+  }
+>;
+
+const tabs = ["Home", "Transactions", "Loans", "Checks", "Liquidity"];
+
+export const Default: TabsStory = ({ width = 600, ...props }) => {
+  return (
+    <div style={{ width, maxWidth: "100%" }}>
+      <TabsNext {...props}>
+        {tabs.map((label) => (
+          <TabPanelNext label={label} key={label}>
+            <Text>Content for {label} tab</Text>
+          </TabPanelNext>
+        ))}
+      </TabsNext>
+    </div>
+  );
+};
+
+export const DefaultSelectedTabIndex: TabsStory = ({
+  width = 600,
+  ...props
+}) => {
+  return (
+    <div style={{ width, maxWidth: "100%" }}>
+      <TabsNext {...props} defaultActiveTabIndex={2}>
+        {tabs.map((label) => (
+          <TabPanelNext label={label} key={label}>
+            <Text>Content for {label} tab</Text>
+          </TabPanelNext>
+        ))}
+      </TabsNext>
+    </div>
+  );
+};
+
+export const WithSecondaryContainer = () => {
+  return (
+    <div
+      style={{
+        width: 600,
+        maxWidth: "100%",
+        height: 300,
+        background: "var(--salt-container-secondary-background)",
+      }}
+    >
+      <TabsNext>
+        {tabs.map((label) => (
+          <TabPanelNext label={label} key={label}>
+            <Text>Content for {label} tab</Text>
+          </TabPanelNext>
+        ))}
+      </TabsNext>
+    </div>
+  );
+};
+
+export const Centered: TabsStory = ({ width = 600, ...props }) => {
+  return (
+    <div style={{ width, maxWidth: "100%" }}>
+      <TabsNext align="center" {...props}>
+        {tabs.map((label) => (
+          <TabPanelNext label={label} key={label}>
+            <Text>Content for {label} tab</Text>
+          </TabPanelNext>
+        ))}
+      </TabsNext>
+    </div>
+  );
+};
+
+export const LotsOfTabs: TabsStory = ({ width = 600, ...props }) => {
+  const tabs = [
+    "Home",
+    "Transactions",
+    "Loans",
+    "Checks",
+    "Liquidity",
+    "With",
+    "Lots",
+    "More",
+    "Additional",
+    "Tabs",
+    "Added",
+    "In order to",
+    "Showcase overflow",
+    "Menu",
+    "On",
+    "Larger",
+    "Screens",
+  ];
+
+  return (
+    <div style={{ width, minWidth: 0, maxWidth: "100%" }}>
+      <TabsNext {...props}>
+        {tabs.map((label) => (
+          <TabPanelNext label={label} key={label}>
+            <Text>Content for {label} tab</Text>
+          </TabPanelNext>
+        ))}
+      </TabsNext>
+    </div>
+  );
+};
+
+export const Controlled: TabsStory = ({ width = 600, ...props }) => {
```

Comment:
would it makes sense to have something external controlling the tabs? or something else that happens with the state change so the difference between cotrolled/uncontrolled examples is visually more evident?

Reply:
Good point